src/model/timestamp.ts

```diff
@@ -1,96 +1,92 @@
+// TODO: use streamId as last part
```

Comment:
Is it a copy of [this file](https://github.com/proxima-one/proxima-npm/blob/master/packages/proxima-streams/src/model/timestamp.ts)? Maybe we can move it to some public package for reusing or even make `proxima-core` public and move it there. @ivanbenko what do you think?

Reply:
For connectivity:
We discussed with Ivan that it's not a problem that the code is copied across repositories, we can leave it as is. It should be considered as a separate code base which interacts only through API so, in theory, even if it diverges it shouldn't break stuff.